sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/models/AnalyzeSentimentAction.java

```diff
@@ -0,0 +1,136 @@
+    private String modelVersion;
+    private boolean includeStatistics;
+    private boolean disableServiceLogs;
+    private boolean includeOpinionMining;
+
+    /**
+     * Get the name of action.
+     *
+     * @return the name of action.
+     */
+    public String getActionName() {
+        return actionName;
+    }
+
+    /**
+     * Set the custom name for the action.
+     *
+     * @param actionName the custom name for the action.
+     *
+     * @return The {@link AnalyzeSentimentAction} object itself.
+     */
+    public AnalyzeSentimentAction setActionName(String actionName) {
+        this.actionName = actionName;
+        return this;
+    }
+
+    /**
+     * Gets the version of the text analytics model used by this operation.
+     *
+     * @return The model version.
+     */
+    public String getModelVersion() {
+        return modelVersion;
+    }
+
+    /**
+     * Set the model version. This value indicates which model will be used for scoring, e.g. "latest", "2019-10-01".
+     * If a model-version is not specified, the API will default to the latest, non-preview version.
+     *
+     * @param modelVersion The model version.
+     *
+     * @return The {@link AnalyzeSentimentAction} object itself.
+     */
+    public AnalyzeSentimentAction setModelVersion(String modelVersion) {
+        this.modelVersion = modelVersion;
+        return this;
+    }
+
+    /**
+     * Get the value of {@code includeStatistics}.
+     *
+     * @return The value of {@code includeStatistics}.
+     */
+    public boolean isIncludeStatistics() {
+        return includeStatistics;
+    }
+
+    /**
+     * Set the value of {@code includeStatistics}. The default value is false by default.
+     * If set to true, indicates that the service should return document and document batch statistics
+     * with the results of the operation.
+     *
+     * @param includeStatistics If a boolean value was specified in the request this field will contain
+     * information about the document payload.
+     *
+     * @return the {@link AnalyzeSentimentAction} object itself.
+     */
+    public AnalyzeSentimentAction setIncludeStatistics(boolean includeStatistics) {
+        this.includeStatistics = includeStatistics;
+        return this;
+    }
+
+    /**
+     * Get the value of {@code disableServiceLogs}.
+     *
+     * @return The value of {@code disableServiceLogs}. The default value of this property is 'false'. This means,
+     * Text Analytics service logs your input text for 48 hours, solely to allow for troubleshooting issues. Setting
+     * this property to true, disables input logging and may limit our ability to investigate issues that occur.
+     */
+    public boolean isServiceLogsDisabled() {
+        return disableServiceLogs;
+    }
+
+    /**
+     * Set the value of {@code disableServiceLogs}.
+     *
+     * @param disableServiceLogs The default value of this property is 'false', except for methods like
```

Comment:
now that we have this property per action type, we don't need to say in which cases the property is true. Consider removing the `except for methods like...`

Reply:
This comment applies to other places